frontends/qt/server/server.go

```diff
@@ -91,6 +92,78 @@ func handleURI(uri *C.char) {
 	bridgecommon.HandleURI(C.GoString(uri))
 }
 
+func matchDarkTheme(themeName string) bool {
+	return strings.Contains(strings.ToLower(strings.TrimSpace(themeName)), "dark")
+}
+
+// detect theme used by OS and return true if it's dark
+func detectDarkTheme() bool {
+	log := logging.Get().WithGroup("server")
+	switch myos := strings.ToLower(runtime.GOOS); myos {
+	case "darwin":
+		cmd := exec.Command("defaults", "read", "-g", "AppleInterfaceStyle")
+		out, err := cmd.Output()
+		if err == nil {
+			log.Info("MacOS theme: " + string(out))
+			if strings.TrimSpace(string(out)) == "Dark" {
+				return true
+			}
+		}
+	case "linux":
+		// Try KDE first, since Kubuntu can also have `gsettings` and that can lead to wrong results
+		cmd := exec.Command("kreadconfig5", "--file", os.ExpandEnv("$HOME/.config/kdeglobals"), "--group", "General", "--key", "ColorScheme")
+		out, err := cmd.Output()
+		if err == nil {
+			log.Info("kde theme: " + string(out))
+			if matchDarkTheme(string(out)) {
+				return true
+			}
+		}
+
+		// Try Gnome/Ubuntu
+		cmd = exec.Command("gsettings", "get", "org.gnome.desktop.interface", "color-scheme")
+		out, err = cmd.Output()
+		if err == nil {
+			log.Info("Gnome/Ubuntu theme: " + string(out))
+			if matchDarkTheme(string(out)) {
+				return true
+			}
+		}
+
+		// Try Cinnamon
+		cmd = exec.Command("gsettings", "get", "org.cinnamon.desktop.interface", "gtk-theme")
+		out, err = cmd.Output()
+		if err == nil {
+			log.Info("Cinnamon theme: " + string(out))
+			if matchDarkTheme(string(out)) {
+				return true
+			}
+		}
+
+		// Try XFCE4
+		cmd = exec.Command("xfconf-query", "-c", "xsettings", "-p", "/Net/ThemeName")
+		out, err = cmd.Output()
+		if err == nil {
+			log.Info("xfce theme: " + string(out))
+			if matchDarkTheme(string(out)) {
+				return true
+			}
+		}
+	case "windows":
+		const regKey = `Software\Microsoft\Windows\CurrentVersion\Themes\Personalize`
```

Comment:
I tested - this has to be 

		const regKey = `HKCU\Software\Microsoft\Windows\CurrentVersion\Themes\Personalize`